providers/provider_default.go

```diff
@@ -105,16 +104,6 @@ func (p *ProviderData) GetLoginURL(redirectURI, state string) string {
 	return a.String()
 }
 
-// CookieForSession serializes a session state for storage in a cookie
```

Comment:
Are these used at all anymore? Or has their usage been replaced by the equivalent `CookieForSession` in utils?

Reply:
Good question, I believe they have pretty much been replaced